diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/BlockStructureBraces.kt

```diff
@@ -217,6 +228,13 @@ class BlockStructureBraces : Rule("block-structure") {
     private fun checkBraceNode(node: ASTNode, shouldContainNewline: Boolean = false) =
             (node.elementType != WHITE_SPACE || shouldContainNewline == node.text.contains("\n"))
 
+    private fun leaveOrAppendOneNewline(node: ASTNode, parent: ASTNode, nodeBefore: ASTNode?){
```

Comment:
Is this method unused after all? Please either use it or remove. If you decide to keep it, it can be done as an extension of `ASTNode`.